docs/source/license.rst

```diff
@@ -1,6 +1,6 @@
-=======
+==============
```

Comment:
adding more `=` to prevent pre-commit thinking this is a merge conflict. Same for other RST files.